### PR TITLE
PEP8 Dataloader filenames

### PR DESCRIPTION
**Pull request by hjabbot**
Simple fix that updates filenames to be pep-8 compliant. Also had to change all the import statements to match the filenames, and also updated the docs to match too 

Here's the reg tests [passing](https://github.com/antarctica/PolarRoute/files/11158127/dl_rename_tests.txt) with the new filenames

### CODE REVIEW

File: polar_route/__init__.py
Comment (Ulvetanna) [inline polar_route/__init__.py:1]: Should this be for `VesselPerformanceModeller` not vessel_factory ?

Comment (SamuelHall700) [reply polar_route/__init__.py:1]: I think you're correct, I've added the VesselPerformanceModeller to the alias as well :)

**Review by Ulvetanna — CHANGES_REQUESTED** [review]
Looks good, but issue in the __init__.py where VesselFactory is defined over `vesselPerformanceModeller`

File: polar_route/__init__.py
Comment (SamuelHall700) [reply polar_route/__init__.py:1]: ^ I've added this in the alias as well

**Review by Ulvetanna — APPROVED** [review]
No issue then.

**Pull request merged**